src/emmetHelper.ts

```diff
@@ -495,7 +495,11 @@ export function isAbbreviationValid(syntax: string, abbreviation: string): boole
 			return false;
 		}
 		if (abbreviation.includes('#')) {
-			return hexColorRegex.test(abbreviation) || propertyHexColorRegex.test(abbreviation);
```

Comment:
I wonder whether `propertyHexColorRegex` should still show up. For example, `bgc:#333` is valid (and expands to `background-color: #333;`).

Reply:
There are test for that case and they still pass
https://github.com/microsoft/vscode-emmet-helper/blob/4fa257726c3c1ceff1b403c21960b2ad4678c2f4/src/test/emmetHelper.test.ts#L713-L721